pkg/liqonet/ipam.go

```diff
@@ -89,8 +89,20 @@ func (liqoIPAM *IPAM) AcquireReservedSubnet(reservedNetwork string) error {
 		return err
 	}
 	if ok && reservedNetwork == pool {
-		klog.Errorf("Network %s is equal to a pool", reservedNetwork)
-		return fmt.Errorf("network %s is equal to a pool so it cannot be reserved", reservedNetwork)
+		klog.Infof("Network %s is equal to a pool, acquiring first half..", reservedNetwork)
+		mask := getMask(reservedNetwork)
```

Comment:
Can you please briefly comment on this logic?

Reply:
Sure!